nautiluszim/entrypoint.py

```diff
@@ -9,7 +9,7 @@
 
 def main():
     parser = argparse.ArgumentParser(
-        prog=NAME, description="Create a ZIM file off a collection of file documents",
```

Comment:
don't change formatting (same for the following)

Reply:
I think the black formatting has changed the formatting. I'll keep the original formatting and change it accordingly.

Reply:
Thank you ; we'll do a dedicated black PR after with checks in the repo to enforce it